### PR TITLE
Db/pose estim

### PR DESCRIPTION
**Pull request by devinbellll**

### CODE REVIEW

**Comment by devinbellll** [comment]
Lets merge this in and delete all the other branches

File: development/applications/hedwig/src/flight_control.c [hunk 1]
```diff
@@ -528,29 +528,31 @@ static void flight_control_task(void *pvParameters)
...
-            PID_alt_D,
-            Alt_Hover_Const,
-            PID_pitch_P,
-            PID_pitch_roll_I,
-            PID_pitch_D,
-            PID_yaw_P,
-            PID_yaw_D);
-
-        rtos_delay_ms(1);
+        // LOG_DEBUG("z: %7.4f dz: %7.4f yaw, pitch, roll: %7.4f, %7.4f, %7.4f p, q, r: %7.4f, %7.4f, %7.4f motors: %u, %u, %u, %u ALT_PD: %7.4f, %7.4f, %7.4f ATT_PID: %7.4f, %7.4f, %7.4f YAW_PD: %7.4f, %7.4f\r\n",
```
Comment (adambujak) [inline development/applications/hedwig/src/flight_control.c:531]: ifdef out instead of comment

File: development/applications/hedwig/src/imu.c [hunk 2]
```diff
@@ -47,7 +51,7 @@ static void imu_calibrate(float *accel_reading, float *gyro_reading)
     // calibration complete
     accel_bias[0] = accel_readings[0] / IMU_CALIBRATION_CYCLES;
     accel_bias[1] = accel_readings[1] / IMU_CALIBRATION_CYCLES;
-    accel_bias[2] = (accel_readings[2] / IMU_CALIBRATION_CYCLES) + 9.80665f;
+    accel_bias[2] = (accel_readings[2] / IMU_CALIBRATION_CYCLES) - 9.80665f;
```
Comment (adambujak) [inline development/applications/hedwig/src/imu.c:54]: what is this number? gravity? add define

File: development/applications/hedwig/src/sensors.c [hunk 3]
```diff
@@ -68,9 +69,16 @@ void sensors_calibrate(void)
...
+{
+  RPY[0]  = atan2(2.0 * (quat[3] * quat[2] + quat[0] * quat[1]) , 1.0 - 2.0 * (quat[1] * quat[1] + quat[2] * quat[2]));
+  RPY[1] = asin(2.0 * (quat[2] * quat[0] - quat[3] * quat[1]));
+  RPY[2]   = atan2(2.0 * (quat[3] * quat[0] + quat[1] * quat[2]) , - 1.0 + 2.0 * (quat[0] * quat[0] + quat[1] * quat[1]));
+}
+
 static void sensors_task(void *pvParameters)
 {
-  LOG_DEBUG("SENSORS TASK STARTED\r\n");
+  LOG_DEBUG("SENSORS TASK STARTED\r\n");
```
Comment (adambujak) [inline development/applications/hedwig/src/sensors.c:81]: no

**Comment by adambujak** [comment]
this is doodoo

**Pull request merged**